### DISCUSSION TITLE
Welcome to Office365 Discussions!

### DISCUSSION THREAD

**Discussion by directorcia** [directorcia/Office365#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by muyiwa12** [comment]
Hello can get a box and what's the rate asap

**Comment by directorcia** [comment]
Sorry??

**Reply by muyiwa12** [reply]
Do you sell company emails (0.365)

**Reply by directorcia** [reply]
No
